lib/luminork-server/Cargo.toml

```diff
@@ -0,0 +1,87 @@
+auth-api-client = { path = "../../lib/auth-api-client" }
+buck2-resources = { path = "../../lib/buck2-resources" }
+dal = { path = "../../lib/dal" }
+edda-client = { path = "../../lib/edda-client" }
+frigg = { path = "../../lib/frigg" }
+module-index-client = { path = "../../lib/module-index-client" }
+nats-multiplexer = { path = "../../lib/nats-multiplexer" }
+nats-multiplexer-client = { path = "../../lib/nats-multiplexer-client" }
+nats-subscriber = { path = "../../lib/nats-subscriber" }
+permissions = { path = "../../lib/permissions" }
+rebaser-client = { path = "../../lib/rebaser-client" }
+sdf-core = { path = "../../lib/sdf-core" }
+sdf-extract = { path = "../../lib/sdf-extract" }
+si-crypto = { path = "../../lib/si-crypto" }
+si-data-nats = { path = "../../lib/si-data-nats" }
+si-data-pg = { path = "../../lib/si-data-pg" }
+si-data-spicedb = { path = "../../lib/si-data-spicedb" }
+si-events = { path = "../../lib/si-events-rs" }
+si-frontend-types = { path = "../../lib/si-frontend-types-rs" }
+si-id = { path = "../../lib/si-id" }
+si-jwt-public-key = { path = "../../lib/si-jwt-public-key" }
+si-layer-cache = { path = "../../lib/si-layer-cache" }
+si-pkg = { path = "../../lib/si-pkg" }
+si-posthog = { path = "../../lib/si-posthog-rs" }
+si-settings = { path = "../../lib/si-settings" }
+si-std = { path = "../../lib/si-std" }
+telemetry = { path = "../../lib/telemetry-rs" }
+telemetry-http = { path = "../../lib/telemetry-http-rs" }
+veritech-client = { path = "../../lib/veritech-client" }
+
+async-openai = { workspace = true }
+async-trait = { workspace = true }
+axum = { workspace = true }
+base64 = { workspace = true }
+chrono = { workspace = true }
+clap = { workspace = true }
+convert_case = { workspace = true }
+derive_builder = { workspace = true }
+derive_more = { workspace = true }
+futures = { workspace = true }
+futures-lite = { workspace = true }
+hyper = { workspace = true }
+itertools = { workspace = true }
+names = { workspace = true }
+nix = { workspace = true }
+once_cell = { workspace = true }
+pathdiff = { workspace = true }
+rand = { workspace = true }
+remain = { workspace = true }
+reqwest = { workspace = true }
+regex = { workspace = true }
+serde = { workspace = true }
+serde_json = { workspace = true }
+serde_with = { workspace = true }
+sodiumoxide = { workspace = true }
+strum = { workspace = true }
+thiserror = { workspace = true }
+tokio = { workspace = true }
+tokio-stream = { workspace = true }
+tokio-tungstenite = { workspace = true }
+tokio-util = { workspace = true }
+tower = { workspace = true }
+tower-http = { workspace = true }
+tracing-tunnel = { workspace = true }
+ulid = { workspace = true }
+utoipa = { workspace = true }
+url = { workspace = true }
+y-sync = { workspace = true }
+
+[dev-dependencies]
+dal-test = { path = "../../lib/dal-test" }
+
+indoc = { workspace = true }
+pretty_assertions_sorted = { workspace = true }
+yrs = { workspace = true }
```

Comment:
I think you can drop these here in and in the BUCK file too since they are for SDF tests.